pkg/autopilot/controller/readyprober.go

```diff
@@ -96,7 +105,7 @@ func (p readyProber) probeOne(target apv1beta2.PlanCommandTargetStatus) error {
 	}
 
 	probe := k8shttpprobe.NewWithTLSConfig(p.tlsConfig, false /* followNonLocalRedirects */)
-	url := fmt.Sprintf(readyzURLFormat, address)
+	url := fmt.Sprintf(readyzURLFormat, address, p.k8sAPIPort)
```

Comment:
Oh, this should probably better be using `net.JoinHostPort`? 🤔

Reply:
Done